e2e/cypress/integration/accessibility/accessibility_dropdowns_spec.js

```diff
@@ -119,7 +119,7 @@ describe('Verify Accessibility Support in Dropdown Menus', () => {
         cy.focused().tab();
 
         // * Verify the accessibility support in the Status Dropdown menu items
-        const labels = ['online', 'away', 'do not disturb. disables all notifications', 'offline'];
+        const labels = [' dialog', 'online', 'away', 'do not disturb. disables all notifications', 'offline'];
```

Comment:
Test is now passing but I suggest to correctly fix the accessibility by:
```diff
--- a/components/status_dropdown/status_dropdown.jsx
+++ b/components/status_dropdown/status_dropdown.jsx
@@ -178,7 +178,7 @@ export default class StatusDropdown extends React.PureComponent {
         return (
             <Menu.Group>
                 <Menu.ItemToggleModalRedux
-                    text={''}
+                    accessibilityLabel='Custom Status'
                     modalId={ModalIdentifiers.CUSTOM_STATUS}
```
```diff
--- a/components/widgets/menu/menu_items/menu_item_toggle_modal_redux.tsx
+++ b/components/widgets/menu/menu_items/menu_item_toggle_modal_redux.tsx
@@ -17,15 +17,16 @@ type Props = {
     dialogProps?: Dictionary<any>;
     extraText?: string;
     text: string;
+    accessibilityLabel: string;
     className?: string;
     children?: React.ReactNode;
     sibling?: React.ReactNode;
 }
 
-export const MenuItemToggleModalReduxImpl: React.FC<Props> = ({modalId, dialogType, dialogProps, text, extraText, children, className, sibling}: Props) => (
+export const MenuItemToggleModalReduxImpl: React.FC<Props> = ({modalId, dialogType, dialogProps, text, accessibilityLabel, extraText, children, className, sibling}: Props) => (
     <>
         <ToggleModalButtonRedux
-            accessibilityLabel={text}
+            accessibilityLabel={accessibilityLabel || text}
             modalId={modalId}
```

Then this should be:
```suggestion
        const labels = ['Custom Status dialog', 'online', 'away', 'do not disturb. disables all notifications', 'offline'];
```

Also, please request for Dev review from webapp team for app code changes.